app/Console/Commands/TransformImports.php

```diff
@@ -103,7 +103,7 @@ public function handle()
                 continue;
             }
 
-            $caption = $ip->caption;
+            $caption = $ip->caption != null ? $ip->caption : "";
```

Comment:
I would consider doing a stricter null check 

```suggestion
            $caption = $ip->caption ?? '';
```